src/FluentCMS.Web.UI/Plugins/Auth/ProfileUpdatePlugin.razor

```diff
@@ -0,0 +1,90 @@
+<Page>
+    <PageHeader HasBack="true"
+                Title="Update Profile">
+    </PageHeader>
+    <PageBody>
+        <Grid Gutter="GridGutter.Medium">
+            <GridItem Small="GridItemColumn.Twelve">
+                <Form Model="@Model"
+                      OnSubmit="OnSubmit">
+                    <Card>
+                        <CardBody>
+                            <Input @bind-Value="Model.Email"
+                                   Id="appUpdateEmailInput"
+                                   Label="Email"
+                                   Name="Email"
+                                   Placeholder="Enter Profile Email..."
+                                   Required="true" />
+                            @*Disabled="true"*@
```

Comment:
@TheHadiAhmadi we need a small fix! can we validate the email field in update profile? and i was not sure about its disabled attribute. can you confirm this?